### DISCUSSION TITLE
Davinci Updates

### DISCUSSION THREAD

**Discussion by rsssxd** [Divested-Mobile/DivestOS-Build#54] [Q&A] [answered]
Davinci will no longer receive updates from LineageOS.
 
https://www.reddit.com/r/LineageOS/comments/su6321/lineageos_171_deprecation_notice/

Can DivestOS continue to provide updates for Davinci? There are dozens of official software such as DotOS that offer updates for Davinci.

**Comment by rsssxd** [comment]
Davinci may receive Android 12 update. But I am not sure.

**Comment by SkewedZeppelin** [comment, marked as answer]
My roster does not depend on what devices Lineage builds for.

**Comment by rsssxd** [comment]
I'm so glad to hear that. I want to continue using DivestOS. Thanks a lot.

**Comment by rsssxd** [comment]
With the last update, several devices got Android 11. Any plans for Mi 9T?

I can test it.
@SkewedZeppelin

**Comment by SkewedZeppelin** [comment]
davinci will get 11 if/when LineageOS officially makes an 11 build for it.
I do not do device porting.

**Comment by rsssxd** [comment]
It probably won't get updates for Android 11. I don't know if it will be Android 12. Lineageos support for Davinci has completely stopped.

**Comment by rsssxd** [comment]
There are unofficial Lineageos builds available for Davinci.

**Comment by rsssxd** [comment]
I verified. Davinci will come again with Android 12 support, it is currently in the testing phase.
